lively.collab/comments/comment.js

```diff
@@ -8,6 +10,7 @@ export class Comment {
     this.timestamp = new Date().getTime();
     this.position = relativePosition;
     this.resolved = undefined;
+    this.username = UserRegistry.current.loadUserFromLocalStorage(config.users.authServerURL).name;
```

Comment:
Does the username identify the a single user? (or can multiple users have the same name)

Reply:
Yes, it does.